mizar/dp/mizar/workflows/builtins/pods/create.py

```diff
@@ -90,5 +93,6 @@ def run(self):
         # Create the corresponding simple endpoint objects
         endpoint_opr.create_simple_endpoints(interfaces, spec)
 
-        # TODO (cathy): in Arktos shall we mark the pod network ready here?
+        if 'arktos.futurewei.com/network' in self.param.body['metadata'].get('labels', {}):
+            endpoint_opr.annotate_builtin_pods(spec['name'], spec['namespace'])
```

Comment:
Have you tested it? Updating the readiness field is what allows Arktos to invoke the CNI. Ideally, We want this to happen after the operator produces the network interface.